Lab_5.ipynb

```diff
@@ -0,0 +1,1121 @@
+      "text/plain": [
+       "<Figure size 900x3000 with 14 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "fig, ax = plt.subplots(len(keys), 1, figsize=(9,30))\n",
+    "fig.tight_layout(pad=1.2)\n",
+    "\n",
+    "for i in range(len(keys)):\n",
+    "    ax[i].hist(background_dict[keys[i]], bins=70, density=True, label=\"Background\")\n",
+    "    ax[i].set_xlabel(str(keys[i]))\n",
+    "    \n",
+    "    ax[i].hist(signal_dict[keys[i]], bins=70, density=True, label=\"Higgs Signal\", alpha=0.7)\n",
+    "    \n",
+    "    ax[i].legend(loc=0)\n",
+    "    \n",
+    "# ax[0,0].set_title(\"Background\", fontsize=20)\n",
+    "# ax[0,1].set_title(\"Higgs Signal\", fontsize=20)\n",
+    "\n",
+    "plt.show()"
```

Comment:
Y-axis labels and titles. Can you separate the plots more to make the X-axis labels more clear which plot they're referring to?